lib/game.rb

```diff
@@ -0,0 +1,79 @@
+require 'pry'
+
+class Game
+
+	attr_accessor :player_1, :player_2, :board
+
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5],  # Middle row
+    [6,7,8],  # bottom row
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [2,4,6],
+    [0,4,8]
+
+  ]
+
+  def initialize (player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+  	@player_1 = player_1
+  	@player_2 = player_2
+  	@board = board
+  end
+
+  def board=(board)
+    @board = board
+  end
+
+  def current_player
+  	board.turn_count.odd? ? player_2 : player_1
+  end
+
+  def over?
+  	draw? || won?
+  end
+
+  def draw?
+  	board.full? && !won?
+  end
+
+  # def won?
+  #   WIN_COMBINATIONS.detect do |combo|
+  #     @board.cells[combo[0]] == @board.cells[combo[1]] && @board.cells[combo[1]] == @board.cells[combo[2]] && @board.cells[combo[0]] != " "
+  #   end
+  # end
+
+  def won?
+    WIN_COMBINATIONS.detect do |win_combination|
+      @board.cells[win_combination[0]] == @board.cells[win_combination[1]] && @board.cells[win_combination[0]] == @board.cells[win_combination[2]] && @board.cells[win_combination[1]] != " "
+    end
+  end
+
+  def winner
+  	board.cells[won?.first] if won?
+  end
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.